src/main/java/uk/gov/companieshouse/api/util/security/Permission.java

```diff
@@ -78,7 +78,12 @@ public enum Key {
         /**
          * Key for company PSC permissions
          */
-        COMPANY_PSCS("company_pscs");
+        COMPANY_PSCS("company_pscs"),
+
+        /**
+         * Key for OE Annual Updates (ROE Update)
+         */
+        COMPANY_OE_UPDATE("company_oe_annual_update");
```

Comment:
Minor comment, is it worth calling this COMPANY_OE_ANNUAL_UPDATE? will there be other updates at some point?

Reply:
Sure - lets make it more specific as it will save a change in the future (if ever needed). Thanks.